app/src/main/java/com/pokerio/app/utils/GameState.kt

```diff
@@ -560,6 +580,44 @@ object GameState {
     fun getMaxBet(): Int {
         return players.maxOf { it.bet }
     }
+
+    fun getSmallBlindPlayer(): Player? {
```

Comment:
I think it would be better to return `Player` instead of `Player?` and add a `check(players.length >= 2)`. If there are less than two players something went wrong anyways

Reply:
When you do this like that then preview will not generate.